Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -506,7 +538,15 @@ extension ActivityStreamPanel: DataObserverDelegate {
     }
 
     func getPocketSites() -> Success {
-        return pocketAPI.globalFeed(items: 4).bindQueue(.main) { pStory in
+        let showPocket = profile.prefs.boolForKey(PrefsKeys.ASPocketStoriesVisible)
+        // Only cancel if the user has created the pref in their profile and it is false
+        // This allows someone who is in an unsupported locale to use Pocket stories
+        if let usersPocketPref = showPocket, !usersPocketPref {
```

Comment:
can's this just be `if !showPocket {`?

Reply:
because showPocket will be false if it is `nil` so if a user hasnt set a pref we still want to make an effort to show them pocket stories (we still need to check their locale).

I agree this is weirdly complicated.